src/instructlab/config/init.py

```diff
@@ -213,7 +221,7 @@ def prompt_user_to_choose_profile(key, arch_family_processors) -> Config | None:
         cfg = read_config(file)
     elif system_profile_selection == 0:
         click.echo(
-            "No profile selected - any hardware acceleration for training must be configured manually."
+            "No profile selected - any hardware acceleration must be configured manually."
```

Comment:
Is it accurate to just say that hardware acceleration must be configured manually?

I would suggest maybe something like 
`No profile selected - ilab will use generic code defaults - these may not be optimized for your system.`

Reply:
that seems like kind of a mouthful, i feel like folks don't care that its the code defaults, but they do care that no acceleration was directly configured in their cfg (gpus: 0)

Reply:
But the configuration being done extends to a lot more than hardware acceleration - I think that's worth calling out, as well as _where_ the values that are being used are coming from (if I don't have a profile, where is my config coming from?)